list_participants_tsv_columns.py

```diff
@@ -0,0 +1,135 @@
+"""List all columns in participants.tsv files in openneuro datasets.
+
+Tries to identify columns:
+- that are all dates or timestamps
+- controlled terms
+- have a description in participants.json
+- columns data type
+- nb of levels in that column
+- nb of rows in the dataset
+
+This is saved in:
+- bulk_annotation_columns.tsv
+
+Also saved:
+- unique_columns.tsv counts the number of for column name
+"""
+
+
+import json
+from pathlib import Path
+from warnings import warn
+
+import pandas as pd
+
+from utils import bulk_annotation_logger, is_euro_format, is_yes_no, output_dir, read_csv
+
+LOG_LEVEL = "INFO"
+
+log = bulk_annotation_logger(LOG_LEVEL)
+
+
+CONTROLLED_TERMS = {
+    "participant_id": "nb:ParticipantID",
```

Comment:
FYI, here are some mappings from human-readable labels to Neurobagel controlled terms that are currently used internally by the bagel-cli: https://github.com/neurobagel/bagel-cli/blob/main/bagel/mappings.py#L22-L30

Based on the contents of `unique_columns.tsv`, for the next PR, we can probably try to reuse some of these mappings (i.e., "age", "sex", "diagnosis") to make the manual annotation step a bit easier.

Reply:
Agreed: I was just having a look at those earlier today.

I think that in the long run we may want to have a repo / package where some of those "utility" functions, mappings can be reused across repo.